app/routes/login/index.tsx

```diff
@@ -15,7 +15,13 @@ export const loader: LoaderFunction = async ({ request }) => {
     return redirect(config.dashboardPath);
   }
 
-  return { devLogin: process.env.DEV_LOGIN === "true", env };
+  return {
+    devLogin: process.env.DEV_LOGIN === "true",
```

Comment:
can we call the booleans with is/has prefix please? isDevLogin, isGithubEnabled etc?